docs/DeveloperGuide.md

```diff
@@ -857,7 +857,18 @@ testers are expected to do more *exploratory* testing.
 
 ## Appendix: Planned Enhancements
 1. The current `newevent` and `edit` commands allows an event's start time to be after its end time. We plan to only allow an event to be added or successfully edited if its start time is before or the same as its end time, with an error message `Start time must be before end time.` being shown otherwise.
+
 2. The current `linkcontact` command does not recognise an inputted phone number if there are extra spaces around it. For example, `linkcontact 1      12345678` is not allowed. We plan to allow the extra spaces by ignoring extra spaces when splitting the input arguments.
+
+3. upon using the current `linkcontact` command on a contact that is already linked to a specified event, the application returns "contact successfully linked" message in the message display box. We plan to change the display message to "Contact already linked" instead for contacts that are already linked.
```

Comment:
Should the messages be in `code format` and have similar capitalisation to be more consistent?

Reply:
good suggestion